docs/src/main/asciidoc/optaplanner.adoc

```diff
@@ -813,6 +813,72 @@ This avoids hard coding a solver time, because the unit test might run on arbitr
 This approach ensures that the test runs long enough to find a feasible solution, even on slow machines.
 But it does not run a millisecond longer than it strictly must, even on fast machines.
 
```

Comment:
This new content should have a separate section title than the TimeTableResource part and it should come first because it's more important to teach users to test their constraints:

Suggestion: Under the section title "Test the application", create 2 subsections:
- "Test the constraints" => the new content
- "Test the solver" => the original content